pyproject.toml

```
@@ -136,18 +136,34 @@ exclude = '''
 profile = 'black'
 skip_gitignore = true
 
-[tool.flake8]
-max-line-length = 99
-doctests = false
-exclude = ["*build/", "docs/"]
-select = "C,E,F,W,B,B950"
-ignore = "N802,N806,W503,E203"
-per-file-ignores = [
-    "*/__init__.py: F401",
-    "docs/conf.py: E265",
-    "/^\\s*\\.\\. _.*?: http/: E501"
+[tool.ruff]
+line-length = 99
+
+[tool.ruff.lint]
+select = [
+  "F",
+  "E",
+  "C",
+  "W",
+  "B",
+]
+ignore = [
+  "N802",
+  "N806",
+  "E203",
 ]
 
+[tool.ruff.lint.flake8-quotes]
+inline-quotes = "double"
+
+[tool.ruff.lint.extend-per-file-ignores]
+"*/__init__.py" = ["F401"]
+"docs/conf.py" = ["E265"]
+"/^\\s*\\.\\. _.*?: http/" = ["E501"]
+
```

I feel it is preferable to avoid constant excludes and use per-file policies. Here, we add C901 only for a particular file where it was found:

```suggestion
"src/eddymotion/estimator.py" = ["C901"]

```